### PR TITLE
Led support

### PR DESCRIPTION
**Pull request by osamuaoki**
Here is a patch to add LED support (+6 bytes) 
I also squeezed code (-6 bytes)
So it is still 506 bytes

I checked and works nicely with LED ;-)

### CODE REVIEW

**Comment by volium** [comment]
Hey @osamuaoki!

I am really sorry I ignored this PR for so long. I have been very busy with other things (mostly work), but I finally have the time to work on nanoBoot once again.

I have already merged @sigprof's PRs (#12 and #13), mainly because they were easier to review and merge.

I want to incorporate your latest changes as well, but this PR is now very hard to follow, and the title ("Led support") no longer reflects all the actual changes being made.

I hope you don't mind, but could you redo the PR so that it is more limited in scope to just adding LED support?

I would guess it makes sense to include a lot of the size optimizations from both you and @sigprof first, so we should probably take care of that prior to merging the LED support itself.

I think I need @sigprof's  help to understand which branches should be merged first (looks like **tmp2** and **string-descriptors**) are further along, but I see some commits are shared with **tmp** branch, so it would be good to know what has been tested that can be integrated back.

I look forward to hearing back from you....

**Comment by osamuaoki** [comment]
Let's keep this PR as is for now until my new PR is tested and reviewed.

**Comment by osamuaoki** [comment]
Since I added clean set of size reduction patches in my lednew branch (PR Lednew #16), I am asking this branch to be cloased.
